### PR TITLE
Fix missing files and remove Litelib

### PR DESCRIPTION
**Pull request by geoperez**
Resolves issue #350

### CODE REVIEW

File: src/EmbedIO.Samples/PeopleController.cs [hunk 1]
```diff
@@ -1,11 +1,10 @@
-﻿using System;
+﻿using EmbedIO.Routing;
```
Comment (rdeago) [inline src/EmbedIO.Samples/PeopleController.cs]: System namespaces first, thanks!

File: src/EmbedIO.Samples/Person.cs [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
         public string EmailAddress { get; set; }
 
 #pragma warning disable 0618 // "Use a better hasher." - Not our fault if gravatar.com uses MD5.
         public string PhotoUrl => $"http://www.gravatar.com/avatar/{Hasher.ComputeMD5(EmailAddress).ToUpperHex()}.png?s=100";
 #pragma warning restore 0618
+
+        internal static async Task<IEnumerable<Person>> GetDataAsync()
+        {
+            // Imagine this is a database call :)
+            await Task.Delay(0);
```
[the remark this answers is not in the document]
Comment (rdeago) [reply src/EmbedIO.Samples/Person.cs]: Can't help but agree with Codacy this time.

File: src/EmbedIO.Samples/PeopleController.cs (hunk 1 above)
Comment (geoperez) [reply src/EmbedIO.Samples/PeopleController.cs]: Done!

File: src/EmbedIO.Samples/Person.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (geoperez) [reply src/EmbedIO.Samples/Person.cs]: Done!

**Pull request merged**